plugins/snap.yaml

```diff
@@ -3,25 +3,23 @@ kind: Plugin
 metadata:
   name: snap
 spec:
-  version: "v0.0.1"
+  version: "v0.1.0"
   homepage: https://github.com/honk-ci/kubectl-snap
   platforms:
   - selector:
       matchExpressions:
       - {key: os, operator: In, values: [darwin, linux]}
-    uri: https://github.com/honk-ci/kubectl-snap/releases/download/0.1/kubectl-snap-0.1.tar.gz
+    uri: https://github.com/honk-ci/kubectl-snap/releases/download/0.1.0/kubectl-snap-0.1.0.tar.gz
     sha256: "8df10f76e82f71b34c737d128b7b369e0daf641118577822872537e787a6755d"
-    files:
-    - from: "*"
-      to: .
     bin: kubectl-snap
-  shortDescription: A chaos test that restores balance to your cluster at the speed of a snap.
+  shortDescription: Randomly delete half of the pods in a namespace or cluster for chaos testing
   caveats: |
+    To reiterate: This will delete half of the pods within the designated space. Currently, if there are pods that have failed or completed, they may get deleted and you may lose debug/log data. 
```

Comment:
Can you clarify how the pods are chosen? For example, does it consider each deployment independently and delete half (rounding?) of the pods in each? Or does it take the set of all pods and randomly delete from those. In the latter case, it may happen that _all_ pods of some deployment get deleted.

In any case, it would be interesting to know if at least one pod of each deployment is guaranteed to survive (again, I'm thinking about rounding errors).

Reply:
> Can you clarify how the pods are chosen? For example, does it consider each deployment independently and delete half (rounding?) of the pods in each? Or does it take the set of all pods and randomly delete from those. In the latter case, it may happen that _all_ pods of some deployment get deleted.

I think this is not too relevant to here.
If users are curious they can read the repo.

Reply:
Well it does make a difference if everything is still working well if at least one pod for each deployment remains. If all pods in a deployment get deleted, this means at least some 503 errors for this service, right?

Reply:
That is a fair point, but I also don't think that merging this in should be blocked by that. The actual goal of the plugin is to delete, at random, half of all pods in a namespace (or all cluster)

I think a flag that would leave at least one pod in a deployment is a decent addition for a 0.2.0 release, even if it somewhat counters the spirit of the **snap**

Reply:
I updated the description to I did update the description to explicitly call out the possibility. I won't resolve this conversation in case there's more to be said.

Reply:
That's cool. As long as users are warned, that's fine.